dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryMessagesTest.java

```diff
@@ -85,14 +88,37 @@ public void testTelemetryMessages() throws Exception {
 
     static void testTelemetryMessages(LibertyServer s, Consumer<List<String>> consoleConsumer) throws Exception {
         String line = s.waitForStringInLog("CWWKF0011I", s.getConsoleLogFile());
-        List<String> linesMessagesLog = s.findStringsInLogs("^(?!.*scopeInfo).*\\[.*$", s.getDefaultLogFile());
-        List<String> linesConsoleLog = s.findStringsInLogs(".*scopeInfo.*", s.getConsoleLogFile());
+        RemoteFile messageLog = s.getDefaultLogFile();
+        RemoteFile consoleLog = s.getConsoleLogFile();
+        s.setMarkToEndOfLog(messageLog);
+        s.setMarkToEndOfLog(consoleLog);
+
+        //Sleep for 5s to ensure messages/console log files are in sync
+        TimeUnit.SECONDS.sleep(5);
+
+        List<String> linesMessagesLog = s.findStringsInLogsUsingMark("^(?!.*scopeInfo).*\\[.*$", messageLog);
+        List<String> linesConsoleLog = s.findStringsInLogsUsingMark(".*scopeInfo.*", consoleLog);
 
         if (consoleConsumer != null) {
             consoleConsumer.accept(linesConsoleLog);
         }
 
-        assertEquals("Messages.log and Telemetry console logs don't match.", linesMessagesLog.size(), linesConsoleLog.size());
+        //Check if number of lines found in messages.log and console.log match.
+        boolean logSizeMatches = (linesMessagesLog.size() == linesConsoleLog.size());
```

Comment:
It would be good to log the numOfLines for each log files, since the new `AssertTrue` would only return a boolean, we won't know what the difference was.